imageio/plugins/tifffile.py

```diff
@@ -73,6 +74,10 @@ class TiffFormat(Format):
     Thus calling :py:meth:`Format.Writer.set_meta_data` after the first frame
     was written has no effect. Also, global metadata is ignored if metadata is
     provided via the `meta` argument of :py:meth:`Format.Writer.append_data`.
+    
+    If you have installed tifffile as a Python package, imageio will attempt 
+    to use that instead of the bundled version from this plugin. Doing so can 
+    provide access to new performance improvements and bug fixes.
```

Comment:
More trailing ws removal.
```suggestion

    If you have installed tifffile as a Python package, imageio will attempt
    to use that instead of the bundled version from this plugin. Doing so can
    provide access to new performance improvements and bug fixes.
```